vortex-datafusion/src/plans.rs

```diff
@@ -143,7 +144,7 @@ impl ExecutionPlan for RowSelectorExec {
             chunk_idx: 0,
             filter_projection: self.filter_projection.clone(),
             conjunction_expr,
-            filter_schema,
+            _filter_schema: filter_schema,
```

Comment:
can we undo this

Reply:
doesn't seem like we currently use it anymore, do you want it for future work or should I just remove it completely?

Reply:
i guess remove for now